spacy/cli/templates/quickstart_training.jinja

```diff
@@ -32,9 +32,12 @@ batch_size = {{ 128 if hardware == "gpu" else 1000 }}
 factory = "transformer"
 
 [components.transformer.model]
-@architectures = "spacy-transformers.TransformerModel.v1"
+@architectures = "spacy-transformers.TransformerModel.v3"
 name = "{{ transformer["name"] }}"
 tokenizer_config = {"use_fast": true}
+transformer_config = {"output_attentions": false}
+mixed_precision = false
+grad_scaler_config = {"init_scale": 32768}
```

Comment:
What are the defaults here? We don't want to add/maintain things here that have good defaults. (And if `false` isn't the default, we need to do something about that.)

Reply:
The defaults are here: https://github.com/explosion/spacy-transformers/blob/master/spacy_transformers/architectures.py#L196, i.e. `false` and empty dict. You're right Adriane it's better to keep the quickstart shorter.